### PR TITLE
Issue #210 Added docs regarding running proxy locally

### LINKED ISSUE #210: jq as dependency
proxy requires jq as a dependency: either add it to the docs, or install it using script or remove it.

### PR DESCRIPTION
**Pull request by kishansagathiya**
- Modified local proxy script to install jq
	- Modified documentation regarding running proxy locally

Fixes #210

### CODE REVIEW

**Comment by kishansagathiya** [comment]
jq works in command line, but for some reason doesn't work well in script.

Through commandline
```
[ksagathi@localhost fabric8-jenkins-proxy]$ oc get pods -l deploymentconfig=jenkins-proxy -o json | jq -r '.items[0].metadata.name'
jenkins-proxy-122-9pmpz
```

If I run script
```

**Comment by kishansagathiya** [comment]
@chmouel Just a small change to documentation

File: README.md [hunk 1]
```diff
@@ -118,8 +118,33 @@ Both CI systems build all merges to master as well as pull requests.
 The repository contains a script [`setupLocalProxy.sh`](./scripts/setupLocalProxy.sh) which can be used to run the Proxy locally.
 A prerequisite for this is access to https://console.rh-idev.openshift.com/.
 To run the script you need to export your OpenShift access token for console.rh-idev.openshift.com as DSAAS_PREVIEW_TOKEN.
-Note, you need edit permissions on the dsaas-preview namespace in order to port forward.
-
+Note, you need edit permissions on the dsaas-preview namespace in order to port forward.
```
Comment (chmouel) [inline README.md]: Note, In order to port forward you need to edit permissions on the dsaas-preview namespace

File: README.md [hunk 2]
```diff
@@ -118,8 +118,33 @@ Both CI systems build all merges to master as well as pull requests.
 The repository contains a script [`setupLocalProxy.sh`](./scripts/setupLocalProxy.sh) which can be used to run the Proxy locally.
 A prerequisite for this is access to https://console.rh-idev.openshift.com/.
 To run the script you need to export your OpenShift access token for console.rh-idev.openshift.com as DSAAS_PREVIEW_TOKEN.
-Note, you need edit permissions on the dsaas-preview namespace in order to port forward.
-
+Note, you need edit permissions on the dsaas-preview namespace in order to port forward. 
+You will also need jq to run these commands. For fedora use `sudo dnf install jq`
```
Comment (chmouel) [inline README.md]: need to have `jq` installed

File: README.md [hunk 3]
```diff
@@ -118,8 +118,33 @@ Both CI systems build all merges to master as well as pull requests.
...
+
+To start proxy and other required services
+```
+[user@localhost ~]$ export DSAAS_PREVIEW_TOKEN=<dsaas-preview-token> 
+[user@localhost ~]$ ./scripts/setupLocalProxy.sh start 
+[user@localhost ~]$ eval $(./scripts/setupLocalProxy.sh env) 
+[user@localhost ~]$ fabric8-jenkins-proxy 
+```
+
+To stop proxy and other services started for running it
```
Comment (chmouel) [inline README.md]: Can you please rephrase here?

Comment (kishansagathiya) [reply README.md]: How about ` To stop running proxy, remove postgres container and stop port-forwarding to prod-preview's idler service and tenant service` ?

Comment (chmouel) [reply README.md]: perfect thanks just add the the in "To stop running *the* proxy, "

**Comment by chmouel** [comment]
tks

**Pull request merged**